mqtt/mqtt-broker/tests/persist_failpoints.rs

```diff
@@ -71,28 +69,23 @@ async fn test_persistor(count: usize, ops: Vec<Op>) {
 
     // clear the failpoints and ensure we can load at least one state
     tear_down_failpoints();
-    let state = persistor.load().await.unwrap();
+    let state = persistor.load().unwrap();
     assert!(state.is_some());
 }
 
 #[test]
 fn test_failpoints_smoketest() {
     let scenario = FailScenario::setup();
-    tokio::runtime::Builder::new()
-        .basic_scheduler()
-        .enable_all()
-        .build()
-        .unwrap()
-        .block_on(async {
-            fail::cfg("filepersistor.load.spawn_blocking", "return").unwrap();
 
-            let tmp_dir = TempDir::new().unwrap();
-            let path = tmp_dir.path().to_owned();
-            let mut persistor = FilePersistor::new(path, ConsolidatedStateFormat::default());
+    fail::cfg("filepersistor.store.fileopen", "return").unwrap();
```

Comment:
This line looks weird. I assume you define that each failpoint to behave as `return`, right? but why only one of failpoints defined here?

Reply:
This test is to make sure that the failpoints are actually enabled. It's to protect against someone disabling the feature in the Config.toml, and us unknowingly switching off the failpoints. I want to make sure that the proptest test is actually enabling failures.

Reply:
I wonder if there is a way to make it in a more obvious way...